cis/schema.json

```diff
@@ -233,15 +243,15 @@
                     "lastUsed": {
                         "type": "string",
                         "format": "date-time",
-                        "description": "UNKNOWN"
+                        "description": "Date at which this group membership last granted access to a resource (generally, an RP) for the user. This may be used to automatically expire and deny access when the lastUsed attribute is older than 3 month, for example."
                         },
                     "name": {
                         "type": "string",
-                        "description": "Name of the group, as given by the IdP."
+                        "description": "Name of the group."
```

Comment:
No - while this structure (`authoritativeGroups`) is present it's not yet used - but we currently do not plan to namespace publishers within it. We had a previous iteration where all groups were namespaced (via a json structure, not a string 'hack'/underscore) but we realized it was currently too unrealistic. This is also why, at this time, only Auth0 is planned to write here.